backend/test/utils.js

```diff
@@ -0,0 +1,14 @@
+import { parse } from '@ipld/dag-json'
+import { chainHeadTestData } from './test_data/chainHead.js'
+import { rawActorEventTestData } from './test_data/rawActorEvent.js'
+
+export const makeRpcRequest = async (method, params) => {
+  switch (method) {
+    case 'Filecoin.ChainHead':
+      return parse(JSON.stringify(chainHeadTestData))
+    case 'Filecoin.GetActorEventsRaw':
+      return parse(JSON.stringify(rawActorEventTestData)).filter(e => e.height >= params[0].fromHeight && e.height <= params[0].toHeight)
+    default:
+      console.error('Unknown method')
```

Comment:
This does not feel right to me. The callers of this stub function will receive `undefined` as the result, I am concerned we will often forget to handle that edge case.

I propose throwing an error instead.

```js
throw new Error(`Unsupported RPC API method: "${method}"`)
```